### DISCUSSION TITLE
Share what you'd love to see in QuickSnip

### DISCUSSION THREAD

**Discussion by technoph1le** [dostonnabotov/quicksnip#48] [Ideas]
Hey, there! 👋

Thank you for taking the time to explore our project. 🚀 We’re always on the lookout for fresh ideas and exciting feature requests to make QuickSnip even better. If you’ve got something awesome in mind, drop it here! 💭✨

We value every suggestion and make it a point to discuss them with our community and maintainers.

Your support means the world to us. 🙌

**Comment by MurtuzaKapasi** [comment]
Try adding some react code snippets

**Reply by technoph1le** [reply]
Now, QuickSnip has support for frameworks and libraries. Feel free to add them.

**Comment by SerJaimeLannister** [comment]
Please try Federating it as per https://github.com/dostonnabotov/quicksnip/issues/127

**Reply by technoph1le** [reply]
We have [plans](https://github.com/dostonnabotov/quicksnip/blob/main/VISION.md) for allowing private snippets with personal accounts. We will definitely look into it.
